### ISSUE TITLE
Сurrent character is always first in the list

### ISSUE THREAD

**Issue by clancyby** [Baganator/Syndicator#35]
**Describe the solution you'd like**
Make an option so that the current character is always first in the list in the hint.

**Comment by wayneluke** [comment]
Would love to see this added.

**Comment by wintark** [comment]
Agreed, most other "what's in my alt's inventory" addons do this, and while I like Syndicator and Baganator a LOT, it's hard to use them without this simple improvement.

**Comment by plusmouse** [comment]
I don't get why you'd want the current character first though. Wouldn't you _want_ to know which other characters have it primarily?

**Comment by wintark** [comment]
For example, if I'm at a vendor with something that costs Artists Acuity (like one of the recipes in the War Within)... if I hover over the currency, I have no idea how much my current character has if they aren't in the top 10 (or whatever I have room to set it to). I can tell if I can afford it, but I can't tell if it's going to eat up all my stock of acuity.
